bleak/backends/corebluetooth/scanner.py

```diff
@@ -87,7 +87,7 @@ async def stop(self):
         except Exception as e:
             logger.warning("stopScan method could not be called: {0}".format(e))
 
-    async def set_scanning_filter(self, **kwargs):
```

Comment:
This is not as critical as `is_connected` in my eyes, but it is still a breaking change.

Reply:
The same workaround I did for `is_connected` won't work here, so we will probably just have to break it.